wlc/main.py

```diff
@@ -86,6 +87,14 @@ def __init__(self, message, detail=None):
         super().__init__(message)
 
 
+class DateTimeEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, datetime):
+            return o.isoformat()
+
+        return json.JSONEncoder.default(self, o)
```

Comment:
```suggestion
        return super().default(self, o)
```
That's probably cleaner way to call this.